.github/workflows/ci.yml

```diff
@@ -71,26 +69,22 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: actions-rs/toolchain@v1
-      - name: ci-cargo-tests
-        run: make ci-cargo-tests
-      - name: ci-tools-install-dependencies
-        run:   |
-               if [ ${{ matrix.os }} == "ubuntu-latest" ]; then
-                    sudo apt-get install libusb-1.0-0-dev
-               elif [ ${{ matrix.os }} == "macos-latest" ]; then
-                    brew install libusb-compat pkg-config
-               else
-                    echo "${{ matrix.os }} not supported"
-                    exit 1
-               fi
-      - name: ci-tools
-        run: make ci-tools
+      - name: ci-job-libraries
+        run: make ci-job-libraries
+      - name: ci-job-archs
+        run: make ci-job-archs
+      - name: ci-job-kernel
+        run: make ci-job-kernel
+      - name: ci-job-chips
+        run: make ci-job-chips
+      - name: ci-job-tools
+        run: make ci-job-tools
 
-  emulation-check:
+  ci-qemu:
```

Comment:
if you change the name of a workflow job, you have to also change the name of the job that bors blocks on in bors.toml